src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java

```diff
@@ -7,13 +7,13 @@
 
 @Data
 public class CreateBookRequestDto {
-    @NotNull
+    @NotNull(message = "can't be null.")
```

Comment:
it looks like these changes are being done in the scope of another PR, aren't they?